use/quickstart.md

```diff
@@ -5,28 +5,171 @@ permalink: /use/quickstart/
+Run your first Beam pipeline to count words in a set of files. 
 
-## Articles & slides
-* [The world beyond batch: Streaming 101](https://www.oreilly.com/ideas/the-world-beyond-batch-streaming-101)
-* [The world beyong batch: Streaming 102](https://www.oreilly.com/ideas/the-world-beyond-batch-streaming-102)
-* [Dataflow/Beam & Spark: A Programming Model Comparison](https://cloud.google.com/dataflow/blog/dataflow-beam-and-spark-comparison)
-* [Dataflow and open source - proposal to join the Apache Incubator](http://googlecloudplatform.blogspot.com/2016/01/Dataflow-and-open-source-proposal-to-join-the-Apache-Incubator.html)
+* TOC
+{:toc}
 
-## Current code
-The following GitHub repositories contain code which will be incorporated into Apache Beam.
 
-* [Dataflow Java SDK](https://github.com/GoogleCloudPlatform/DataflowJavaSDK)
-* [Flink Dataflow runner](https://github.com/dataArtisans/flink-dataflow)
-* [Spark Dataflow runner](https://github.com/cloudera/spark-dataflow)
+## Set up your Development Environment
+ 
+1. Download and install the [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) version 1.7 or later. Verify that the [JAVA_HOME](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars001.html) environment variable is set and points to your JDK installation.
 
-These code repositories will be refactored and managed together (along with other code and new contributions) into a single repository.
+1. Download and install [Apache Maven](http://maven.apache.org/download.cgi) by following Maven's [installation guide](http://maven.apache.org/install.html) for your specific operating system.
 
-## Documentation
-* [Apache Beam incubation proposal](https://goo.gl/KJrEl7)
-* *Apache Beam technical vision*
-    * [Detailed](https://goo.gl/5qZt3d)
-    * [Summary](https://goo.gl/nk5OM0)
-* [Apache Beam technical documentation](https://goo.gl/ps8twC)
+
+## Get the WordCount Code
+
+The easiest way to get a copy of the WordCount pipeline is to use the following command to generate a simple maven project that contains Beam's WordCount examples and builds against the most recent Beam release: 
+
+```
+$ mvn archetype:generate \
+      -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-examples \
+      -DarchetypeVersion=LATEST \
+      -DarchetypeGroupId=org.apache.beam \
+      -DgroupId=org.example \
+      -DartifactId=word-count-beam \
+      -Dversion="0.1" \
+      -DinteractiveMode=false \
+      -Dpackage=org.apache.beam.examples
+```
+
+This will create a directory `word-count-beam` that contains a simple pom file and series of example pipelines that count words in text files. 
```

Comment:
the WordCount (with backticks) which count words in text files.
I think just WordCount is sufficient since we described it earlier about it counting words in files.